pages/guilds.md

```diff
@@ -51,33 +53,32 @@ work, and make a formal announcement if the guild is created.
 
 All guilds will have a sponsor on the leadership team. The sponsor is not
 responsible for setting the objectives for a guild, but will check in to
-ensure a guild is setting objectives and holding itself accountable.
+ensure a guild is holding itself accountable to its goals.
 
 Guild sponsors will convene one meeting each quarter to evaluate guild
 progress. Based on the outcome of this meeting, the sponsor may recommend
 spinning down a guild or revoke sponsorship altogether. If a guild has its
-sponsorship revoked, it may continue to operate as a working group, but will
-no longer receive mentorship and advocacy from a sponsor.
+sponsorship revoked, its charter may need to be revised to remove project team
+requirements and may lose dedicated resources, however, it may continue to operate as a working group.
 
 ## <a name="privileges-and-responsibilities"></a>Privileges and Responsibilities
 
 Sponsorship implies that guilds have a voice in discussions amongst the 18F
 leadership team. This in turn implies that guilds may more easily influence
-the objectives of the broader team than working groups. The price of this
-representation is a responsibility for setting and meeting objectives that are
-in service to organizational priorities.
+the objectives of the broader team than working groups. Collaboration with
+the guild sponsor allows for aligning guild objectives with 18F organizational priorities.
 
-Broadly speaking, guild responsibilities include:
+Guild responsibilities include:
 
-* Establishing [standards and/or best practices](http://18f.github.io/guides/)
+* Establish a Charter
```

Comment:
Nit: Reads better to me if these start with gerunds (e.g. "Establishing"), but not worth changing unless you agree.